ui/pages/connected-accounts/connected-accounts.component.js

```diff
@@ -1,75 +1,122 @@
 import PropTypes from 'prop-types';
-import React, { PureComponent } from 'react';
+import React from 'react';
+import { WALLET_SNAP_PERMISSION_KEY } from '@metamask/snaps-utils';
+import { SnapCaveatType } from '@metamask/rpc-methods-flask';
```

Comment:
```suggestion
import { SnapCaveatType } from '@metamask/rpc-methods';
```

Reply:
You should also be able to import it from `@metamask/snaps-utils`

Reply:
Nice catch! Updated